Library/UpdateBackingInput+Constructor.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+import KsApi
+
+extension UpdateBackingInput {
+  internal static func input(
+    from updateBackingData: UpdateBackingData
+  ) -> UpdateBackingInput {
+    let pledgeAmount: String? = updateBackingData.pledgeAmount.flatMap { pledgeAmount in
+      pledgeAmountString(withAmount: pledgeAmount, shippingRule: updateBackingData.shippingRule)
+    }
+    let backingId: String = updateBackingData.backing.graphID
+    let locationId: String? = updateBackingData.shippingRule.flatMap { $0.location.graphID }
+    let rewardId: String? = updateBackingData.reward.graphID
+
+    return UpdateBackingInput(
+      amount: pledgeAmount,
+      applePay: nil,
+      id: backingId,
+      locationId: locationId,
+      paymentSourceId: nil,
+      rewardId: rewardId
+    )
+  }
+}
+
+func pledgeAmountString(withAmount amount: Double, shippingRule: ShippingRule?) -> String {
```

Comment:
Will move this to a shared place once we implement the new `CreateBackingInput`. Also not sure if we're going the route of splitting out amount and shipping rule for that.